lib/conditionals.md

```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is an element that controls the flow of the code. A conditional evaluates a given expression to be true or false. If true, the code following the conditional statement will continue to run. For example, for "if 2 + 2 == 4", the condition will be true because the expression "2 + 2 = 4" is true. 
```

Comment:
You said:

> A conditional is an element...

"Element" is a word that gets used elsewhere in programming. "Statement" would be more appropriate.